docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -373,6 +373,37 @@ paths:
         - developerAPIKey: []
 
   /groups/{id}:
+    get:
+      tags:
+        - groups
+      summary: Fetch a group
+      description: >
+        Any request (including unauthenticated requests) may retrieve any publicly-readable group. When using AuthClient credentials, it is additionally possible to retrieve any group within the associated authority.  Requests authenticated with an API key are able to retrieve any publicly-readable group as well as any private group that the authenticated user created or is a member of.
```

Comment:
Is it possible to wrap this over multiple lines for easier reading?

Reply:
Sure!